com.unity.formats.alembic/Editor/Importer/AlembicImporterEditor.cs

```diff
@@ -397,6 +407,141 @@ static Material LoadMaterial(string assetName, string[] searchPath)
 
             return AssetDatabase.LoadAssetAtPath<Material>(path);
         }
+
+        void DrawHairUI(AlembicImporter importer, bool isMultiEdit)
```

Comment:
the 2nd parameter ` isMultiEdit ` is not used

Reply:
removed in a2f5565